### ISSUE TITLE
Screen reader is not announcing the place holder text in the edit field

### ISSUE THREAD

**Issue by Swati1700** [microsoft/vscode#138114]
### **Actual:**
The screen reader is not announcing the place holder text "Select a Language" when focus goes on the edit field .

Similar issue is observed in the below scenario
**Home - Select a template for your projects first function**
1. Open the application
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button
4. Click on "Create New Project Button" from the toolbar
5. Start the screen reader and observe the announcement by screen reader about "Select a template for your projects first function".

### **Expected:**
The screen reader should announce the place holder text "Select a Language" when focus goes on the edit field .

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#Win10;#VisualStudioCode-Win32-Apr2021;#Visual Studio Code Client;#FTP;#DesktopApp;#NVDA;#JAWS;#WCAG1.3.1;#Element:EditField;

### **Environment Details:**
Application Name: Azure Tools for VS Code
Windows Version: Win10
Screen reader: NVDA2020.4, JAWS 2021

### **Repro Steps:**
1. Open the application.
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button
4. Click on "Create New Project Button" from the toolbar
5. Start the screen reader and observe the announcement by screen reader about "Select a Language".

### **User Impact:**
Screen reader users will not get correct information present on the screen.

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!

**Comment by jinglouMSFT** [comment]
@Swati1700 @isidorn I have put a similar comment in the linked bug  https://github.com/microsoft/vscode-azurefunctions/issues/2937

First of all, I couldn't repro this issue using NVDA 2021.2. Watermark strings in both the Functions extension and in the VS Code Git:Clone commend were read as expected. Below are the details. 
 
In the Functions extension, I have tried this scenario on Windows 10, "Select a language" was read by NVDA. Below is a capture of what was read: 
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/10081483/144148821-0f437860-1591-4779-89ea-41e2572edde7.png)

A similar experience using VS Code with the Git:Clone commend, which uses basically the same UI component. Below was what was read for the Git:Clone comment (without needing any of the extensions installed to repro)
![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/10081483/144149100-ae63ce22-57cc-4c4f-bbdc-0d71af553404.png)
The text that should be read is "Provide repository URL or pick a repository source".

The behavior between the Functions extension and the VS Code itself is consistent.   Both watermark strings are read by NVDA 2021.2.

**Comment by Rama8345** [comment]
Verified the issue at Azure Tools for VS Code application. The issue is fixed. So, closing the bug. please find attachment for reference. Thanks
**Github Tags:**#Closed;

![138114_Closed](https://user-images.githubusercontent.com/115216656/220571104-0810e886-3823-4b01-b681-4f8dd869a698.gif)
